util/null.go

```diff
@@ -37,9 +37,16 @@ func IntPtrToInt(ptr *int64) int64 {
 	return *ptr
 }
 
-func IntPtrToString(ptr *int64) int64 {
+func Int32PtrToInt(ptr *int32) int32 {
```

Comment:
This function wasn't used, and also not doing what it's advertising, so I reused and converted the one we actually use